script.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+// form issue 해결
+// submit이벤트가 발생하면 브라우저가refresh된다.
+// e.preventDefault()를 통해서 브라우저의 refresh를 막고
+// 새로운 변수 location에 input값 저장해서 textContent로 출력
+const Form = document.querySelector("form");
+const inputBox = document.querySelector("input");
+const waitingList = document.querySelector("#waitingList");
```

Comment:
```suggestion
const waitingList = document.querySelector("#waiting-list");
```

css classname/id 는 `kebab-case`로 짓는 것이 일반적이랍니다!